### ISSUE TITLE
Optimize the UI for Big Sur

### ISSUE THREAD

**Issue by sindresorhus** [ImageOptim/ImageOptim#329]
There's nothing wrong with the current UI, but it could be slightly optimized for Big Sur. I mocked this up in SwiftUI in a few minutes. Just an idea. Feel free to reject.

<img width="776" alt="Screen Shot 2020-12-03 at 20 45 40" src="https://user-images.githubusercontent.com/170270/101025887-c45ae080-35a8-11eb-8a9a-b91067a551b2.png">

It would still have the drop symbol in the middle of the window. I didn't bother to add that.

I would also shorten the summary.

---

For reference, the current UI:

<img width="775" alt="Screen Shot 2020-12-03 at 20 45 50" src="https://user-images.githubusercontent.com/170270/101025922-d177cf80-35a8-11eb-946e-64dbb3d0bf4e.png">

**Comment by kornelski** [comment]
That looks much more Apple-modern indeed!

Do you think SwiftUI is actually usable enough for thisd? 

The current UI code is pretty old, as old as Mac OS X Panther, so I wouldn't mind rewriting it.

**Comment by sindresorhus** [comment]
Definitely. SwiftUI doesn't currently support multi-column table views. That's the only obstacle. But it's possible to [wrap AppKit components in SwiftUI](https://github.com/search?q=language:swift+nsviewrepresentable+nstableview&type=code). Alternatively, go with some other kind of list layout.

**Comment by naher94** [comment]
I wonder if its also worth updating the "drop symbol" as well using the SF Symbols, might be a nice touch

**Comment by mrleblanc101** [comment]
Would make sens to update to app icon the the "Squircle" iOS-esque shape

**Comment by mrleblanc101** [comment]
What about Liquid Glass 🙊
